includes/options/class-amp-options-manager.php

```diff
@@ -562,6 +571,63 @@ public static function persistent_object_caching_notice() {
 		}
 	}
 
+	/**
+	 * Render the Stories deprecation admin notice.
+	 */
+	public static function render_stories_deprecation_notice() {
+		if (
+			self::is_stories_experience_enabled() &&
+			(
+				'edit-amp_story' === get_current_screen()->id ||
+				'toplevel_page_' . self::OPTION_NAME === get_current_screen()->id
+			)
+		) {
+			echo '<div class="notice notice-warning"><p>';
+			printf(
+				'%s %s %s',
+				esc_html__( 'The Stories experience is being extracted from the AMP plugin into a separate standalone plugin which will be available soon. Please backup or', 'amp' ),
+				sprintf(
+					'<a href="%s">%s</a>',
+					esc_url( 'https://amp-wp.org/documentation/amp-stories/exporting-stories/' ),
+					esc_html__( 'export your existing Stories', 'amp' )
+				),
+				esc_html__( 'as they will not be available in the next version of the AMP plugin.', 'amp' )
+			);
+			echo '</p></div>';
+		} elseif ( ! self::is_stories_experience_enabled() && 'toplevel_page_' . self::OPTION_NAME === get_current_screen()->id ) {
+			printf(
+				'<div class="notice notice-info"><p>%s</p></div>',
+				esc_html__( 'The Stories experience has been removed from the AMP plugin. This beta feature is being split into a separate standalone plugin which will be available for installation soon.', 'amp' )
+			);
+		}
+	}
+
+	/**
+	 * Render the Stories deprecation notice in the Story editor.
+	 */
+	public static function render_stories_deprecation_editor_notice() {
+		$script = sprintf(
+			"( function( wp ) {
+						wp.data.dispatch( 'core/notices' ).createNotice(
+							'warning',
+							%s,
+							{
+						  		isDismissible: false,
+						  		actions: [
+									{
+										url: 'https://amp-wp.org/documentation/amp-stories/exporting-stories/',
+										label: 'View how to export your Stories',
```

Comment:
This needs to be translatable.

```suggestion
										label: <?php echo wp_json_encode( __( 'View how to export your Stories', 'amp' ) ) ?>,
```